krux/__init__.py

```diff
@@ -1 +1 @@
-VERSION = '3.1.2'
+VERSION = '3.2.0'
```

Comment:
Nit: minor version? patch version?

Reply:
I think those flags disappearing, which conforms to expected behavior but was not happening in reality, is considered enough to not be a patch version.